src/css/pagination.scss

```diff
@@ -1,59 +1,48 @@
 .wrapper {
   display: flex;
+  gap: 0.5rem;
+  align-items: center;
   list-style-type: none;
   margin-bottom: 20px;
 
   .paginationItem {
-    border: 1px solid rgb(0, 0, 0, 0.6);
-    height: 34px;
-    width: 34px;
-    border-radius: 50%;
-    overflow: hidden;
-    margin: 5px;
-    position: relative;
-
-    &:hover {
-      background-color: rgba(159, 159, 159, 0.295);
-      cursor: pointer;
-    }
-
+    display: flex;
     button {
-      display: flex;
+      border: 1px solid rgb(0, 0, 0, 0.6);
+      background: transparent;
+      height: 34px;
+      width: 34px;
+      border-radius: 50%;
+      overflow: hidden;
       color: rgb(0, 0, 0);
-      background-color: transparent;
-      border: none;
-      outline: none;
 
       &:hover {
+        background-color: rgba(159, 159, 159, 0.295);
         cursor: pointer;
       }
-    }
 
-    .arrowButton {
-      width: 100%;
-      height: 100%;
-      padding: 12px;
-    }
-
-    .itemButton {
-      position: absolute;
-      top: 50%;
-      left: 50%;
-      transform: translateX(-50%) translateY(-50%);
+      &:disabled {
+        pointer-events: none;
+        opacity: 0.5;
+      }
     }
 
-    &.selected {
-      background-color: rgba(159, 159, 159, 0.295);
-
+    &[aria-current="page"] {
```

Comment:
Replaced `.disabled` with `&:disabled` and `.selected` with `&[aria-current="page"]`.